flink-ml-api/pom.xml

```diff
@@ -38,6 +38,34 @@ under the License.
       <version>${flink.version}</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-table-api-java-bridge_${scala.binary.version}</artifactId>
```

Comment:
One minor question here, why do we need these dependencies?

Reply:
Thanks for the review Zhipeng. You are right that `flink-table-api-java-bridge` and `flink-java` are not needed here. The `flink-table-planner` and `flink-test-utils` can be changed to use `test` scope since they are only needed by unit tests (e.g. PipelineTest).

I will update the PR to fix this issue.

Reply:
The PR has been updated to fix this issue.